### PR TITLE
[DO NOT MERGE] Edits to External participants in Yammer

### PR DESCRIPTION
**Pull request by jdmartinez36**
Made requested edits to the article.

Added an intro sentence to the body to describe why an admin would want this list. I leveraged it from add-external-participants.md,  but the author should verify that it accurately describes the reason.

Moved content in step 2 to a note above the image, and indented info and the image that's directly part of step 2.

Minor edits for clarity.

@tiburd

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/work-with-external-users/find-external-participants.md [hunk 1]
```diff
@@ -21,24 +21,27 @@ description: "Yammer admins can use data export to find external participants in
...
     You'll only see this option if you are a verified admin in the Yammer network.
-    
+
     For more information, see [Export Yammer Enterprise data](../manage-security-and-compliance/export-yammer-enterprise-data.md).
-    
-2. To identify **threads in your network that users from other networks participate in**, locate the export folder on your computer, and open the **MessageThreads.Outbound.csv** export file. 
+
+2. To identify **threads in your network that users from other networks participate in**, locate the export folder on your computer, and open the **MessageThreads.Outbound.csv** export file.
+
+    >[!NOTE]
```
Comment (JohanFreelancer9) [inline yammer/work-with-external-users/find-external-participants.md:37]:
```suggestion
    > [!NOTE]
```

File: yammer/work-with-external-users/find-external-participants.md [hunk 2]
```diff
@@ -21,24 +21,27 @@ description: "Yammer admins can use data export to find external participants in
...
+    >The data export reflects the current view of the network. For example, if a user was added, but then removed before the report was created, that user won't appear in the report.
```
Comment (JohanFreelancer9) [inline yammer/work-with-external-users/find-external-participants.md:38]:
```suggestion
    > The data export reflects the current view of the network. For example, if a user was added, but then removed before the report was created, that user won't appear in the report.
```

**Comment by kwekuako** [comment]
> Copy review - just two minor styling edits.

Is this PR ready to merge? Title says do not merge.

**Pull request merged**